From 70bab632bcde8be801b2e7691115a728cdbf59ce Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Mon, 2 Oct 2023 14:02:21 +0200
Subject: [PATCH] wget: Do not autoreconf

Autoreconfiguring wget can result in broken setup with
GNULIBHEADERS_OVERRIDE_WINT_T not being set. This might be caused by
something being too old, haven't dug deep enough to find the root cause,
but for now using upstream generated configure and Makefile will help us
keep going.
---
 net/wget/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/net/wget/Makefile b/net/wget/Makefile
index f70c0def7..0d1556ba3 100644
--- a/net/wget/Makefile
+++ b/net/wget/Makefile
@@ -20,7 +20,6 @@ PKG_LICENSE:=GPL-3.0-or-later
 PKG_LICENSE_FILES:=COPYING
 PKG_CPE_ID:=cpe:/a:gnu:wget
 
-PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
 PKG_BUILD_PARALLEL:=1
 
-- 
2.42.0

